packages/keyboard-keys/src/index.ts

```diff
@@ -1,2 +1,4 @@
-// TODO: replace with real exports
-export {};
+// API extractor does not support `export * as` syntax
+// https://github.com/microsoft/rushstack/issues/2780
+// export * as keyCodes from './keyCodes';
```

Comment:
I think @andrefcdias you will have the same problem in #18778

Any help for this one @Hotell  @layershifter ?

Reply:
this is a well known limitation of ES modules heh.

2 suggestions:
- use `export as namespace` 
  - not sure if this would tree shake properly (in native environment it should probably?) 
-  add `Code` suffix to all constants
  - while this will be a breaking change, I don't think there is other option (also users can quite easily do a search/replace)

Reply:
looks like we might have a winner ! :D 

![2021-07-12 at 11 46](https://user-images.githubusercontent.com/1223799/125267220-e4087300-e306-11eb-8efb-bf2c965530d9.png)

Reply:
ah I miss-read the original "issue" sorry @ling1726 ! so we cannot use export namespace because of API Extractor limitation.

so 

- add Code suffix to all constants (Cancel -> CancelCode)
   - while this will be a breaking change, I don't think there is other option (also users can quite easily do a search/replace)
- turn off api exctractor for this one temporarily ? 
  - ping folks from rush to try to implement this sooner than later
  -  we dont wanna get blocked by a tool that doesn't support standard ECMA language features as a tax of providing bad ergonomics to our user

Reply:
As linked in the PR description: microsoft/rushstack#2780

Namespace export is not compatible with API extractor.... @Hotell  suggested offline to disable api-extractor for this package

Reply:
also "manual re-export works" in API-Extractor, but that might not work with tree shaking?

```
import * as controls from './controls';
export { controls }
```

Reply:
@Hotell  after #18920 works like a charm